release_tool_utils.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Adding this concerned me. We are appending `msg` in this file. I changed it to String.new and I believe that allows for appending. Check this file for any other usages where frozen string might bite us.

Reply:
The rest of this file looks safe to me.